<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Evan Teran</author>
 <class>BinaryInfo::DialogHeader</class>
 <widget class="QDialog" name="DialogHeader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Header Explorer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Regions To Search:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtSearch"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Results:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>Header</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-close"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset theme="help-browser"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExplore">
       <property name="text">
        <string>Explore Header</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtSearch</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>btnHelp</tabstop>
  <tabstop>btnExplore</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogHeader</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
